src/archivematicaCommon/lib/elasticsearch/aipfile_mets_mapping.json

```diff
@@ -0,0 +1,474 @@
+{
+  "properties": {
+    "dmdSec": {
+      "properties": {
+        "ns0:xmlData_dict_list": {
+          "properties": {
+            "@xmlns:ns1": {
+              "type": "string"
+            },
+            "@xmlns:ns0": {
+              "type": "string"
+            },
+            "@xmlns:xsi": {
+              "type": "string"
+            },
+            "ns1:dublincore_dict_list": {
+              "properties": {
+                "dc:provenance": {
+                  "type": "string"
+                },
+                "dc:date": {
+                  "type": "string"
+                },
+                "@xsi:schemaLocation": {
+                  "type": "string"
+                },
+                "dc:title": {
+                  "type": "string"
+                },
+                "dc:description": {
+                  "type": "string"
+                }
```

Comment:
Maybe the SIP-level DC from the form had all the fields, but the metadata.csv that created the file-level DC didn't have all the columns?  Do we index both SIP & file-level DC in aip/aipfile?  (Do we want to index both?)